### ISSUE TITLE
prisma deploy does not deploy, but generates a docker-compose.yml file

### ISSUE THREAD

**Issue by marktani** [prismagraphql/prisma#2578]
# Bug Report

**Current behavior**

**prisma version 1.8.3**
`prisma deploy` does not deploy, but generates a `docker-compose.yml` file instead.

**prisma version 1.9.0-beta.8**
cannot connect to server

**Reproduction**

```yml
# .graphqlconfig.yml
projects:
  app:
    schemaPath: src/schema.graphql
    extensions:
      endpoints:
        default: http://localhost:4000
  database:
    schemaPath: src/generated/prisma.graphql
    extensions:
      prisma: database/prisma.yml
```

```graphql
# ./database/datamodel.graphql
type Post {
  id: ID! @unique
  createdAt: DateTime!
  updatedAt: DateTime!
  isPublished: Boolean! @default(value: "false")
  title: String!
  text: String!
  author: User!
}

type User {
  id: ID! @unique
  email: String! @unique
  password: String!
  name: String!
  posts: [Post!]!
}
```

```graphql
# ./database/seed.graphql
mutation {
  createUser(data: {
    email: "developer@example.com"
    password: "$2a$10$hACwQ5/HQI6FhbIISOUVeusy3sKyUDhSq36fF5d/54aAdiygJPFzm" # plaintext password: "nooneknows"
    name: "Sarah"
    posts: {
      create: [{
        title: "Hello World 👋"
        text: "I like turtles."
        isPublished: true
      }, {
        title: "Join us at GraphQL Europe 🇪🇺 "
        text: "Get a 10%-discount with this promo code on graphql-europe.org: gql-boilerplates"
        isPublished: true
      }, {
        title: "Solving world hunger"
        text: "This is a draft..."
        isPublished: false
      }]
    }
  }) {
    id
  }
}
```

```yml
# ./database/prisma.yml
# The endpoint of your Prisma API (deployed to a Prisma Sandbox).

# The file containing the definition of your data model.
datamodel: datamodel.graphql

# Seed your service with initial data based on `seed.graphql`.
seed:
  import: seed.graphql

# Download the GraphQL schema of the Prisma API into
# `src/generated/prisma.graphql` (as specfied in `.graphqlconfig.yml`).
hooks:
  post-deploy:
    - graphql get-schema --project database

# If specified, the `secret` must be used to generate a JWT which is attached
# to the `Authorization` header of HTTP requests made against the Prisma API.
# Info: https://www.prisma.io/docs/reference/prisma-api/concepts-utee3eiquo#authentication
# secret: ${env:PRISMA_SECRET}

endpoint: localhost:4466/node/dev
```

Now run `prisma deploy`. This is the result:

**CLI version 1.8.3**

![image](https://user-images.githubusercontent.com/1780597/40983745-92fef432-68e0-11e8-8e05-c1fc6c88ca3d.png)

**CLI version 1.9.0-beta.8**

![image](https://user-images.githubusercontent.com/1780597/40984045-41bf274e-68e1-11e8-8974-3fba0e1508f0.png)

**Additional information**
![image](https://user-images.githubusercontent.com/1780597/40983810-af82421c-68e0-11e8-9ad4-7085d0cedb28.png)

**Expected behavior?**

The expected behaviour is that the service `node` in stage `dev` will be deployed to the local cluster at `localhost:4466`.

**Comment by samandeggs** [comment]
Is there any way to disable the `docker-compose.yml` generation feature altogether? We're hitting a similar but unrelated issue.

**Comment by marktani** [comment]
Could you please open a new issue detailing your specific problem, @samandeggs? 🙂

**Comment by marktani** [comment]
1) the error that happens for `1.8.3` was fixed in `1.9-beta` already
2) the error that happens for `1.9-beta` can be fixed by changing the endpoint to `http://localhost:4466/node/dev`. I'll open a separate issue for improving the error message.
